src/test/rustdoc/issue-32556.rs

```diff
@@ -1,5 +1,5 @@
 /// Blah blah blah
-/// ```ignore (testing rustdoc's handling of ignore)
+/// ```ignore (test rustdocs handling of ignor)
```

Comment:
I don't think you meant to change this.
```suggestion
/// ```ignore (testing rustdoc's handling of ignore)
```